CHANGELOG.md

```diff
@@ -5,15 +5,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Legend
+# Legend
```

Comment:
Should stay `##` and have a newline after the header.

Reply:
Fixed